e2e-tests/README.md

```diff
@@ -0,0 +1,189 @@
+End-to-End Tests for Socorro
+=============================
+
+Continuous Integration
+----------------------
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.stage.saucelabs.svg?label=stage)](https://webqa-ci.mozilla.com/job/socorro.stage.saucelabs/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/socorro.prod.saucelabs.svg?label=prod)](https://webqa-ci.mozilla.com/job/socorro.prod.saucelabs/)
+
+
+This directory holds Socorro client-based end-to-end tests which is why they're different than the rest of the code in this repository.
+
+To review the specific Python packages the tests utilize, please review `e2e-tests/requirements.txt`.
+
+Set up and run Socorro tests
+-----------------------------
+
+Review the documentation for [pytest-selenium][pytest-selenium] and decide which browser
+environment you wish to target.
+
+We suggest using a different virtual environment for these tests than the
+rest of Socorro so you're not mixing requirements:
+
+	$ mkvirtualenv socorro-tests
+	$ pip install -r requirements.txt
+
```

Comment:
Because these have the potential of breaking the virtualenv people use for socorro, I think you should add a warning here that it's important to **not re-use the socorro virtualenv**.